tensorflow_addons/losses/tests/general_loss_test.py

```diff
@@ -0,0 +1,387 @@
+# Copyright 2019 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""Tests for General Loss."""
+
+
+import numpy as np
+import tensorflow as tf
+from tensorflow_addons.losses import (
+    general_loss,
+    GeneralLoss,
+)
+
+
+def testConfig():
+    bce_obj = GeneralLoss(reduction=tf.keras.losses.Reduction.NONE, name="general_loss")
+    assert bce_obj.name == "general_loss"
+    assert bce_obj.reduction == tf.keras.losses.Reduction.NONE
+
+
+def assertAllClose(a, b, rtol=1e-15, atol=1e-15):
+    np.testing.assert_allclose(a, b, rtol=rtol, atol=atol)
+
+
+def assertAllEqual(a, b):
+    np.testing.assert_array_equal(a, b)
+
+
+def assertTrue(a):
+    assert a == True  # noqa
```

Comment:
I would suggest we don't need to use wrapper functions. Directly calling `np.testing.assert_*` or `assert` is enough and straightforward.